manim/scene/vector_space_scene.py

```diff
@@ -79,20 +79,90 @@ def add_plane(self, animate: bool = False, **kwargs):
         self.add(plane)
         return plane
 
-    def add_axes(self, animate: bool = False, color: bool = WHITE, **kwargs):
+    def add_axes(
```

Comment:
### Why did I remove the color argument?
Because it is not working. 
`self.add_axes(color=RED)` will not work, because the color of the VGroup is set before the axes creation. 
I could add these lines:
```py
if color:
    axes.set_color(color)
```
but I don't see the reason, the much more powerful would be the new solution when a user could use dictionaries to specify the exact all parameters in axes